pipeline/filters/aws-metadata.md

```diff
@@ -16,21 +16,27 @@ Note: _If you run Fluent Bit in a container, you may have to use instance metada
 
 ### Metadata Fields
 
-Currently, the plugin only adds the instance ID and availability zone. AWS plans to [expand this plugin in the future](https://github.com/fluent/fluent-bit/issues/1780).
+Currently, the plugin support 8 type of EC2 metadata.
 
 | Key | Value |
 | :--- | :--- |
 | az | The [availability zone](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html); for example, "us-east-1a". |
 | ec2\_instance\_id | The EC2 instance ID. |
+| ec2\_instance\_type  | The EC2 instance type. |
+| private\_ip | The EC2 instance private ip. |
+| ami\_id | The EC2 instance image id. |
+| account\_id | The account ID for current EC2 instance. |
+| hostname | The hostname for current EC2 instance. |
+| vpc\_id | The VPC ID for current EC2 instance. |
```

Comment:
If you look at some of the other plugin docs, they have a column for the default value for each option- can you add that in a new PR.

Reply:
Yeah sure, opened a new [PR](https://github.com/fluent/fluent-bit-docs/pull/392) for this